Bio/PDB/__init__.py

```diff
@@ -72,3 +81,10 @@
     from .NeighborSearch import NeighborSearch
 except ImportError:
     pass
+
+# Native Shrake-Rupley algorithm for SASA calculations.
+# Depends on kdtrees C module
+try:
+    from .SASA import ShrakeRupley
```

Comment:
Why import this here? Admittedly Bio.PDB has a lot of historic automatic import, but both the tests and the NEWS entry suggest you'd use:

```python
from Bio.PDB.SASA import ShrakeRupley
```

Reply:
For simplicity, I can remove it of course!